compose/production/django/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN chmod +x /entrypoint
 COPY --chown=django:django ./compose/production/django/start /start
 RUN sed -i 's/\r$//g' /start
 RUN chmod +x /start
-COPY --chown=django:django . /app
+COPY --chown=django:django ./pra_request_tracker /app
```

Comment:
We only need the things inside of `pra_request_tracker` to run the app and if we don't do this then all the import paths will be messed up anyway.